test_common/page_objects/et1/representatives_details_page.rb

```diff
@@ -90,6 +90,24 @@ def set(*args); field.set(*args); end
             element :field, :css, 'input'
             def set(*args); field.set(*args); end
           end
+          section :representative_contact_preference, '.representative_contact_preference' do
+            include ::EtFullSystem::Test::I18n
+            section :email_preference, :form_labelled, 'Email' do
```

Comment:
hard coding strings like this will break the multi lingual testing - try running it in welsh and you will see what i mean.  This is why we did all those 'translated' capybara selectors so that you can specify a ruby symbol (which it then knows means you want it translated - that was the way we decided a long time ago)